flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -155,6 +155,24 @@ SqlNodeList TableProperties():
     {  return new SqlNodeList(proList, span.end(this)); }
 }
 
+SqlCreate SqlCreateFunction(Span s, boolean replace) :
+{
+    SqlIdentifier functionName = null;
+    SqlCharStringLiteral functionClassName = null;
+}
+{
+    <FUNCTION>
+
```

Comment:
Do we need to add 'IF NOT EXIST'' syntax here?

Reply:
Good catch. Added.